app/src/main/java/de/westnordost/streetcomplete/quests/opening_hours/parser/OpeningHoursParser.kt

```diff
@@ -71,6 +92,16 @@ fun List<Rule>.isSupported(): Boolean =
     // anyway, it is not representable directly by SC
     (!weekdaysCollideWithAnother())
 
+/** Returns true if the collection times are supported by StreetComplete
+ * Returns false otherwise, in cases where it is not directly representable */
+@JvmName("isSupportedCollectionTimesRuleList")
+fun List<Rule>.isSupportedCollectionTimes(): Boolean =
+    // all rules must be supported
+    all { it.isSupportedCollectionTimes() } &&
+        // this kind of collection times specification likely require fix
+        // anyway, it is not representable directly by SC
+        (!weekdaysCollideWithAnother())
+
 fun Rule.isSupported(): Boolean =
```

Comment:
since they are now two different "isSupported", it'd be better to name the other one accordingly too - i.e. `isSupportedOpeningHours` etc